app/src/main/java/org/mifos/mobilebanking/ui/activities/LoginActivity.java

```diff
@@ -63,6 +64,8 @@ protected void onCreate(Bundle savedInstanceState) {
     @Override
     public void onLoginSuccess(String userName) {
         this.userName = userName;
+        PreferencesHelper preferencesHelper = new PreferencesHelper(this);
```

Comment:
@jaindiv26 Don't create Instance like this, use Dagger Annotation Inject to do the same.